source/stdafx.h

```diff
@@ -41,8 +41,8 @@ enum CLEO_Version
 
     CLEO_VERSION = CLEO_VER_4,
 };
-#define VERSION_LONG		0x04031700
-#define VERSION_STRING	"4.3.23.0"
+#define VERSION_LONG		0x04032302
+#define VERSION_STRING	"4.3.23.2"
```

Comment:
@JuniorDjjr can you please bump the version to 4.3.24? Less numbers is easier for end-users to remember.

Reply:
Okay, I just didn't want to disturb the official version count, because I already launched an .23 unofficially some time ago and I didn't want to jump to .24 while officially is .22.

Reply:
This was updated now, and only this.